.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "nuget"
+    directory: "/"
```

Comment:
💭 This may need to be a full path.

Reply:
Might be. I have never used it for .NET before.
Let's try and see